### DISCUSSION TITLE
What training data was used?

### DISCUSSION THREAD

**Discussion by lassoan** [ENHANCE-PET/MOOSE#158] [General]
Thank you for sharing these models! I'm wondering if it could make sense to make them available in 3D Slicer.

How your models compare to TotalSegmentator and MONAI Auto3DSeg models?

What data sets have you trained on? The paper that is recommended for citation talks about a 50-100 training images.

**Comment by LalithShiyam** [comment]
Hi @lassoan
We would be delighted to do it! It's just that we don't know how to do it! Moose can be used as a package, may be it makes things easier? We are open to suggestions. 
Moose v1 that came out 3 years ago (published) had 40 datasets. The current version is trained on 1700 datasets approximately (we are planning to release the datasets as well, we are dealing with IRB issues). Our datasets are mostly whole body oncology cases as we are nuclear medicine folks, we are continuously adding more cases but this time we are focusing on ortho and trauma cases. Integrating with slicer would be amazing for us! 
Cheers,
Lalith

**Comment by Keyn34** [comment]
Hello @lassoan, we managed to incorporate MOOSE into 3DSlicer and opened a PR: https://github.com/Slicer/ExtensionsIndex/pull/2127

Thank you for the suggestion and excellent work.

**Reply by lassoan** [reply]
Thank you, nice work, added some comments, but there is nothing major, so it should go through the review quickly.
